Handler/Feed.hs

```diff
@@ -24,10 +24,10 @@ feedFromComments (Entity siteId site) comments = do
     render <- getUrlRender
 
     rssFeedText Feed
-        { feedAuthor      = siteRssAuthor site
-        , feedTitle       = siteRssTitle site
-        , feedDescription = siteRssDescription site
-        , feedLanguage    = siteRssLanguage site
+        { feedAuthor      = siteName site
+        , feedTitle       = "Comments on " <> siteName site
+        , feedDescription = toHtml $ "Comments on " <> siteName site
```

Comment:
```
λ> toHtml "Foo & Bar"
Foo &amp; Bar
```

I used the `String` instance in this test, because the `Text` one wasn't in scope. I'm sure they behave the same w.r.t. escaping.